.github/workflows/go-lint.yml

```diff
@@ -20,13 +20,11 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version-file: ./go.mod
-      - run: CODEGEN_VERIFY=1 make gen-cue
       - run: make gen-go
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.60.1
+          version: v1.62.0
           args: |
-            --config .golangci.toml --max-same-issues=0 --max-issues-per-linter=0 --verbose $(./scripts/go-workspace/golangci-lint-includes.sh)
-          skip-cache: true
+            --verbose $(go list -m -f '{{.Dir}}' | xargs -I{} sh -c 'test ! -f {}/.nolint && echo {}/...')
```

Comment:
i like the `.nolint` approach 🎉. should this change happen in the [Makefile](https://github.com/grafana/grafana/blob/356d7d3045ce60cc6336aaadeb000594c65a0fc4/Makefile#L12) as well? if so, feel free to remove the [go.work comments](https://github.com/grafana/grafana/blob/287ce2ef40a2b5465cfff24afb8014d1d55eef7b/go.work#L3) and related shell script

Reply:
Ideally, yes, I'd like rely on `go list` + `.nolint` rather than on custom scripts. But I'm not sure if there is anything else apart from our Makefile, Drone and GHA pipelines that uses the go.work comments + script to filter the workspaces.

Since this PR already does quite a few changes at once, I'd probably postpone Makefile change and go.work cleanup until we're sure that it works well on the CI.

Reply:
👍 makes sense to me